### PR TITLE
1 create pages

### LINKED ISSUE #1: Create Pages
#### Create the pages:
- Home
- Pricing
- About
- Profile 

Follow [Figma design](https://www.figma.com/file/XRfOpZ3TuPMPQTa43x3PtG/Kayaks-la-isla?type=design&node-id=0%3A1&mode=dev&t=C7qrD2GCQcC3h7MW-1)

### PR DESCRIPTION
**Pull request by Metsanpeitto**
#### Create the pages:
- Home
- Pricing
- About
- Profile 

Follow [Figma design](https://www.figma.com/file/XRfOpZ3TuPMPQTa43x3PtG/Kayaks-la-isla?type=design&node-id=0%3A1&mode=dev&t=C7qrD2GCQcC3h7MW-1)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,101 @@
...
+
+
+
+### Live
+
+
+### Setup
+
+```cmd
+git clone  https://github.com/Metsanpeitto/kayaks_next.git
```
Comment (samuelneff) [inline README.md]: nit
```suggestion
git clone https://github.com/Metsanpeitto/kayaks_next.git
```

File: README.md [hunk 2]
```diff
@@ -1 +1,101 @@
...
+
+```cmd
+npx eslint . --fix
+npx stylelint "**/*.{css,scss}" --fix
+```
+
+## To run jest test
+```cmd
+npm run test
+npm run jest
```
Comment (samuelneff) [inline README.md]: Two? difference?

File: README.md [hunk 3]
```diff
@@ -1 +1,101 @@
...
+```
+
+If you are non a technical person you can download the zip file.
```
Comment (samuelneff) [inline README.md]: A little condescending.
```suggestion
You can download the zip file.
```

File: README.md [hunk 4]
```diff
@@ -1 +1,101 @@
...
+
+- GitHub: [@metsanpeitto](https://github.com/Metsanpeitto)
+- LinkedIn: [Abraham Rodriguez](https://www.linkedin.com/in/abraham-rodriguez-3283a319a/)
+- Portfolio: [Waldenberg](https://portfolio.waldenberginc.com)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (samuelneff) [inline README.md]: Use absolute link, not relative. Relative won't work when looking at the README locally.

File: README.md [hunk 5]
```diff
@@ -1 +1,101 @@
...
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+[![License: CC BY-NC 4.0](https://licensebuttons.net/l/by-nc/4.0/80x15.png)](https://creativecommons.org/licenses/by-nc/4.0/)
+[![License: CC0-1.0](https://licensebuttons.net/l/zero/1.0/80x15.png)](http://creativecommons.org/publicdomain/zero/1.0/)
```
Comment (samuelneff) [inline README.md]: Why two?

File: app/About/page.tsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+//import Card from "./components/Card"
+import styles from "./product.module.css";
+
+export default async function AboutPage() {
+  return (
+    <div className={styles.page}>
+      <h1 className="">About</h1>
```
Comment (samuelneff) [inline app/About/page.tsx]:
```suggestion
      <h1>About</h1>
```

File: app/fonts.ts [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import { Lobster, Roboto } from "next/font/google";
+
+export const lobster = Lobster({
```
Comment (samuelneff) [inline app/fonts.ts:3]: 😎 Cool

File: app/page.tsx [hunk 8]
```diff
@@ -1,95 +1,19 @@
...
-          <p>
-            Instantly deploy your Next.js site to a shareable URL with Vercel.
-          </p>
-        </a>
-      </div>
+      {/*<NavBar />*/}
+      <Image
+        src={heroImage}
+        width={500}
+        height={500}
```
Comment (samuelneff) [inline app/page.tsx:11]: 👍 Good but two comments.

1. You might run into people that object to putting the size here saying you don't need it. That's very old thinking (and something I just learned with Qwik within the last week). Be prepared to talk about why it's important to include the size.
2. If the size matches the actual size of the image, look if there are any npm modules that can insert the image for you automatically and get the size of the image at compile time.
3. That said, the image should be bigger than the specified size so screens with a higher DPI (which is almost all screens at this point) can display it cleaner than the logical DPI.

File: package.json [hunk 9]
```diff
@@ -11,9 +11,12 @@
     "format:fix": "prettier --write --list-different ."
   },
   "dependencies": {
+    "@next/font": "^14.1.4",
+    "classnames": "^2.5.1",
     "next": "14.1.4",
     "react": "^18",
-    "react-dom": "^18"
+    "react-dom": "^18",
+    "typed-css-modules": "^0.9.1"
```
Comment (samuelneff) [inline package.json]: Two comments.

1. `typed-css-modules` should be a dev dependencies.
2. You have a mix of dependencies using `^` and using exact versions. Be consistent and use `^` everywhere.

**Review by samuelneff — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Metsanpeitto) [reply README.md]: - [x] Done

File: README.md (hunk 2 above)
Comment (Metsanpeitto) [reply README.md]: They aren't really needed for now. 
i remove both.

File: README.md (hunk 3 above)
Comment (Metsanpeitto) [reply README.md]: Mea Culpa.
Removed

File: README.md (hunk 4 above)
Comment (Metsanpeitto) [reply README.md]: These lines weren't needed.
Removed

File: README.md (hunk 5 above)
Comment (Metsanpeitto) [reply README.md]: No real reason. This is 3 years old code that I use as a template.
Removed.

File: app/About/page.tsx (hunk 6 above)
Comment (Metsanpeitto) [reply app/About/page.tsx]: [x] Fixed

File: package.json (hunk 9 above)
Comment (Metsanpeitto) [reply package.json]: Ok
[x] Done

**Pull request merged**